msi/release.sh

```diff
@@ -63,9 +63,13 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
 EOF
 echo "$LIST" |
 tr / \\\\ |
-sed 's/\(.*\)\\\(.*\)/            <Component Directory="INSTALLFOLDER:\\\1\\">\
+sed -e 's/\(.*\)\\\(.*\)/            <Component Directory="INSTALLFOLDER:\\\1\\">\
+                <File Source="&" \/>\
+            <\/Component>/' \
+	    -e 's/^\([^\\]*\)$/            <Component Directory="INSTALLFOLDER">\
                 <File Source="&" \/>\
             <\/Component>/'
+
```

Comment:
Why redo this commit, with completely different authorship, when I already did it: dscho/build-extra@306e63f2d6c8b83a79c2614602abad60a7df3231 (I see that you even partially copied the commit message, but you left out the last paragraph that was intended to help reviewers by not forcing them to guess why the `Guid` was skipped).